projects/transformers/run_utils.py

```diff
@@ -141,6 +153,12 @@ def evaluate_tasks(trainer, output_dir, tasks, eval_datasets):
                     logging.info(f"  {key} = {value}")
                     writer.write(f"{key} = {value}\n")
 
+    # if you want drop_last for training, this toggles it back on
+    if drop_last:
+        trainer.args.dataloader_drop_last = True
+        logging.info("Switched trainer.args.dataloader_drop_last"
```

Comment:
I think we should lower the level of this logging, logging.debug should be enough. There is no useful information to the user here, it is more of a print for debugging purposes.

Reply:
Got it. Those were mainly to verify that my fix was working properly, not needed going forward. I removed the bit above and a couple related ones.